css/css-color/parsing/META.yml

```diff
@@ -447,15 +17,32 @@ links:
         - test: color-valid.html
         - test: color-mix-invalid.html
         - test: color-invalid.html
-        - test: color-mixed-num-pct.html
+        - test: color-invalid-hwb.html
+        - test: color-valid-rgb.html
+        - test: color-valid-color-function.html
+        - test: color-valid-hsl.html
+        - test: color-valid-hwb.html
+        - test: color-computed-color-mix-function.html
+        - test: color-computed-hsl.html
+        - test: color-computed-hwb.html
+        - test: color-invalid-lab.html
+        - test: color-valid-color-mix-function.html
+        - test: color-valid-lab.html
+        - test: color-computed-color-function.html
+        - test: color-computed-rgb.html
+        - test: color-invalid-hsl.html
+        - test: color-invalid-rgb.html
+        - test: color-computed-lab.html
+        - test: color-invalid-color-function.html
+        - test: color-invalid-color-mix-function.html
```

Comment:
Looks like you left in some old filenames? (Having these sorted would probably be useful, too.)